tools/compile-contract.py

```diff
@@ -0,0 +1,31 @@
+import sys
```

Comment:
Similarly here I don't think it makes sense to commit to the public repo unless there is some documentation on how to use it.

Reply:
I think it's fixed